src/core/server/http/http_server.test.ts

```diff
@@ -1431,3 +1434,71 @@ describe('setup contract', () => {
     });
   });
 });
+
+describe('Graceful shutdown', () => {
+  test('it should wait until the ongoing requests are resolved and reject any new ones', async () => {
+    const gracefulShutdownTimeout = config.gracefulShutdownTimeout.asMilliseconds();
+
+    const { registerRouter, server: innerServer } = await server.setup(config);
+
+    const router = new Router('', logger, enhanceWithContext);
+    router.post(
+      {
+        path: '/',
+        validate: { body: schema.object({ test: schema.number() }) },
```

Comment:
Do you use `test` param anywhere?

Reply:
It came from the copy-paste from the previous test 😅 
I'll remove it.